data/soapvalidator/Folders/Folder-Retention-Policy.xml

```diff
@@ -159,7 +159,7 @@
             </t:select>
         </t:response>
     </t:test>
-
+   <t:delay sec="60"/>
```

Comment:
60s is too much, any other reasonable value?

Reply:
I have tried 30, 40 second as well but its working intermittently.
With 60 sec delay it always paases.

Reply:
Hmm.. 60Sec is bad for product if any but its fine according to your analysis.